blueprints/semantic-ui-ember/index.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
   afterInstall: function () {
-    return this.addBowerPackageToProject('semantic-ui');
+    return this.addPackageToProject('semantic-ui');
```

Comment:
we can remove the entire blueprint itself and add dependency to package.json

Reply:
Great idea ! I will refactor this weekend